apps/KeePassXC/install-64

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
-version=2.6.6-5
+version=2.6.6-1
 
-install_packages https://apt.raspbian-addons.org/debian/pool/main/k/keepassxc/keepassxc_${version}_arm64.deb || exit 1
+if [ "$(get_codename)" == "bullseye" ]; then
+    #install bullseye version
+    install_packages https://apt.raspbian-addons.org/debian/pool/main/k/keepassxc/keepassxc_${version}_arm64.deb || exit 1
+else 
+    #install buster version
+    install_packages https://github.com/ryanfortner/ryanfortner/releases/download/1002/keepassxc_2.6.6-5_arm64.deb || exit 1
```

Comment:
@ryanfortner, do you want a version variable added to this URL?

Reply:
no, there isn't any way to get the latest version number from ryanforner's github so there isn't any reason to specify it with a variable. They made one release a long time ago and then just update/add files to it as necessary. https://github.com/ryanfortner/ryanfortner/releases its clear if you look at the page